### PR TITLE
APOLLO-17514 crawl db fields

### PR DESCRIPTION
**Pull request by puneetkhanal**
crawldb documentation fields

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -59,5 +59,37 @@ The gRPC framework provides flexibility in the way services and their methods ar
 The data persisted by the connectors framework is distributed across the Fusion cluster. Each node holds its primary partition of the data, as well as backups of other partitions.
 If a node goes down during a crawl, the data store remains intact and usable. Connector implementations do not need to be concerned with this layer, because it is all handled by Fusion.
 
+== CrawlDB fields
+* Core fields required for any connector would include: id and state_s.
+
+* Connector specific values include the "fields" and "metadata" properties, which result in Solr document prefixed fields: field_ and meta_, respectively.
```
Comment (roblucar) [inline README.asciidoc]: Looks good. Are there any "field_" examples we can include that are common across all connectors like "body" or "title"?

Comment (mwmitchell) [reply README.asciidoc]: Hmm, there might be a `lastModified` field now? @wsolaligue, does that new field get indexed into crawl-db using `field_*`?

Comment (wsolaligue) [reply README.asciidoc]: @mwmitchell yes, all the entries on candidates' field map are indexed with the prefix `field_`, but if you mean the field used by the purge stray items, that one is `modifiedAt_l` without the `field_` prefix

File: README.asciidoc [hunk 2]
```diff
@@ -59,5 +59,37 @@ The gRPC framework provides flexibility in the way services and their methods ar
...
+| targetPhase_s  | Name of the phase this item is emitted to.  | content  
+| sourcePhase_s | Name of the phase an item was emitted from.  | content  
+| isTransient_b  | Flag to indicate that the item should be removed from CrawDB after it has been processed.  | false  
+| isLeafNode_b  | This flag is used to prioritize the processing leaf node instead of nested nodes to avoid emitting of too many Candidates.  | false  
+| createdAt_l  | Item created timestamp.  | 1566508663611  
+| createdAt_tdt  | Item created ISO date.  | 2019-08-22T21:17:43.611Z  
+|  modifiedAt_l |  Item modified timestamp. | 1566508665709  
+| modifiedAt_tdt  | Item modified ISO date.  | 2019-08-22T21:17:45.709Z  
+| fetchInput_id_s  | FetchInput Id.  | /app 
+| meta_lastModified_l  | Last modified timestamp for directory/file.  | 1566485811000
```
Comment (acesar) [inline README.asciidoc]: the `meta_` are per/dependent on the plugin - it shouldn't be part of this list, right?

File: README.asciidoc [hunk 3]
```diff
@@ -59,5 +59,37 @@ The gRPC framework provides flexibility in the way services and their methods ar
...
+| meta_name_s  | Name of directory/file. | app  
+| meta_length_l  | Content lenght of the file | 4096  
+| meta_owner_s  | Owner of directory/file.  | root  
+| meta_group_s  | Group of directory/file.  | root  
+| meta_path_s  |  Absolute path of the directory/file. | /app  
+| meta_fetchCounter_l  | Number of items fetched.  | 1 
+| meta_isFile_b  | Is the item file?  | false  
+| meta_exists_b  | Directory/file exits or not?  | true 
+| meta_isDirectory_b  | Is the item directory?  | true  
+| \_version_ | Solr generated version number | 1642603392772603905
```
Comment (acesar) [inline README.asciidoc]: maybe we shouldn't mention the Solr specific fields.

File: README.asciidoc (hunk 2 above)
Comment (puneetkhanal) [reply README.asciidoc]: I have removed meta fields now.

File: README.asciidoc (hunk 3 above)
Comment (puneetkhanal) [reply README.asciidoc]: I have removed solr specific fields now.

File: README.asciidoc (hunk 1 above)
Comment (acesar) [reply README.asciidoc]: @puneetkhanal  In the `modifiedAt_l` documentation there is no mention of the  purge stray items feature that Willy was mentioning. Should we add a note about it?

Comment (mwmitchell) [reply README.asciidoc]: @wsolaligue Is the modifiedAt field part of content documents only? Or are they also on crawl-db docs? If only content docs, let's include the description of that in a separate PR (content doc fields vs crawl-db doc fields).

Comment (wsolaligue) [reply README.asciidoc]: @mwmitchell `modifiedAt_l` is only part of the crawldb-docs, this field is not propagated to the content docs

**Pull request merged**